tests/end_to_end/test_brute_force/test_brute_force_ssh/test_brute_force_ssh.py

```diff
@@ -0,0 +1,66 @@
+import os
+import json
+import re
+import pytest
+from tempfile import gettempdir
+
+from wazuh_testing.tools import configuration as config
+from wazuh_testing import end_to_end as e2e
+from wazuh_testing import event_monitor as evm
+
+# Test cases data
+alerts_json = os.path.join(gettempdir(), 'alerts.json')
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+test_cases_file_path = os.path.join(test_data_path, 'test_cases', 'cases_brute_force.yaml')
+
+# Playbooks
+events_playbooks = ['generate_events.yaml']
+teardown_playbooks = None
+
+# Configuration
+configurations, configuration_metadata, cases_ids = config.get_test_cases_data(test_cases_file_path)
+
+
+@pytest.mark.filterwarnings('ignore::urllib3.exceptions.InsecureRequestWarning')
+@pytest.mark.parametrize('metadata', configuration_metadata, ids=cases_ids)
+def test_brute_force(metadata, get_dashboard_credentials, generate_events, clean_alerts_index):
```

Comment:
```suggestion
def test_brute_force_ssh(metadata, get_dashboard_credentials, generate_events, clean_alerts_index):
```

Reply:
Done in 123ef4c40e7ce10616c8601a498cd48decfe62e7